snow/networking/router/chain_router_test.go

```diff
@@ -1543,6 +1597,7 @@ func TestValidatorOnlyAllowedNodeMessageDrops(t *testing.T) {
 		"",
 		prometheus.NewRegistry(),
 	))
+	defer chainRouter.Shutdown(context.TODO())
```

Comment:
I don't think we're ever planning on having a more canonical context in the tests - so normally we just use `Background`.
```suggestion
	defer chainRouter.Shutdown(context.Background())
```
(I think this applies a few times to this file)